packages/siopv2-oid4vp-rp-auth/src/agent/SIOPv2RP.ts

```diff
@@ -173,6 +175,22 @@ export class SIOPv2RP implements IAgentPlugin {
     )
   }
 
+  private async siopImportDefinitions(args: ImportDefinitionsArgs, context: IRequiredContext): Promise<void> {
+    const { definitions, tenantId, version, versionControlMode } = args
+    await Promise.all(
+      definitions.map((definition) => {
+        console.log(`persisting definition ${definition.id} / ${definition.name} with versionControlMode ${versionControlMode}`)
```

Comment:
debug?

Reply:
Is useful to have in the agent log. I usually don't have debug mode on in the agents